src/legacy/components/Spinner.jsx

```diff
@@ -36,7 +36,7 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import spinnerImg from 'pc-nrfconnect-shared/resources/ajax-loader.gif';
+import spinnerImg from 'pc-nrfconnect-shared/src/Dialog/ajax-loader.gif';
```

Comment:
As the spinner component is also provided by shared, can't that be exported from here?

Reply:
I want to be free to change the components in shared to make them fit the new design without worrying whether I break apps that still use the old design. Because of that I prefer not to use the Spinner component for apps still showing the legacy design.